### PR TITLE
Move wait-for-rabbit functionality into runit's check script

### PR DESCRIPTION
**Pull request by srenatus**
Runit allows for adding check scripts that are executed to see if the
service is up and running. That way, after `sv start rabbitmq` (or
`/opt/opscode/init/rabbitmq start`) has finished successfully, we know
the service is up.

This removes the need for the wait-for-rabbit script.

Initially, it became apparent in #1220 that there are still situations
where rabbitmq has been started, but is not ready yet.

Signed-off-by: Steven Danna <steve@chef.io>
Signed-off-by: Stephan Renatus <srenatus@chef.io>

### CODE REVIEW

**Comment by stevendanna** [comment]
Also fixes: https://github.com/chef/chef-server/issues/1213

File: omnibus/files/private-chef-cookbooks/private-chef/templates/default/sv-rabbitmq-check.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+/usr/bin/env ERL_EPMD_ADDRESS=127.0.0.1 HOME=/root /opt/opscode/embedded/bin/rabbitmqctl wait "<%= node['private_chef']['rabbitmq']['data_dir'] %>/<%= node['private_chef']['rabbitmq']['nodename'] %>.pid"
```
Comment (marcparadise) [inline omnibus/files/private-chef-cookbooks/private-chef/templates/default/sv-rabbitmq-check.erb]: I don't think I see this template referenced anywhere.

Comment (stevendanna) [inline omnibus/files/private-chef-cookbooks/private-chef/templates/default/sv-rabbitmq-check.erb]: @marcparadise This is some magic of the runit cookkbook, like the other sv- templates.

Comment (marcparadise) [inline omnibus/files/private-chef-cookbooks/private-chef/templates/default/sv-rabbitmq-check.erb]: Ah nevermind, I see the runit cookbook uses it

**Comment by srenatus** [comment]
Started an internal build/ci run, will merge if this makes it through :)

**Comment by srenatus** [comment]
It made it through build and test 🎉 Merging.

**Pull request merged**